### PR TITLE
update USAFacts urls

### PR DESCRIPTION
**Pull request by smcclure17**
updates the urls for the USAFacts csvs. Reading the csvs directly into pandas (eg: `pd.read_csv(url)`) threw a 403 forbidden error, so this adds a formal request with a fake user agent to bypass this

### CODE REVIEW

File: can_tools/scrapers/usafacts/data.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 from typing import Any
 
 import pandas as pd
-
+from urllib.request import Request, urlopen
```
[the remark this answers is not in the document]
Comment (smcclure17) [reply can_tools/scrapers/usafacts/data.py:4]: I think these should be used in [lines 32 and 33](https://github.com/covid-projections/can-scrapers/pull/351/files#diff-3136552d691963762219ab40a0d49a2e19e39d4ae60207cab8f2f1909ba35691R32-R33)

**Comment by smcclure17** [comment]
> 
> Do we need to update this for deaths updater as well?
 
The file extensions/names did not change, only the base of the URL, so this works for both

File: can_tools/scrapers/usafacts/data.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ghop02) [reply can_tools/scrapers/usafacts/data.py:4]: ah sorry - good call

**Comment by ghop02** [comment]
> > Do we need to update this for deaths updater as well?
> 
> The file extensions/names didn't change, only the base of the URL did, so this works for both

Ah makes sense :) a bit of a funny design that the deaths updater inherits the cases updater instead of a base updater class, but no need to change that now

**Comment by smcclure17** [comment]
true, that definitely would be cleaner! Going to merge if everything's all set, and thanks for taking a look 🙏

**Pull request merged**